tests/testthat/test-utils.R

```diff
@@ -0,0 +1,28 @@
+# Load Sample Databases ----
+db_data_classic <- readRDS(system.file("testdata/db_data_classic.RDS", package = "REDCapTidieR"))
+db_metadata_classic <- readRDS(system.file("testdata/db_metadata_classic.RDS", package = "REDCapTidieR"))
+
+test_that("multi_choice_to_labels works", {
+  out <- multi_choice_to_labels(db_data = db_data_classic,
+                                db_metadata = db_metadata_classic)
+
+  # Test general structure
+  expect_true(is.data.frame(out))
+  expect_true(nrow(out) > 0)
+
+  # Test multichoice options return expected values and datatypes
+  expect_logical(out$yesno)
+  expect_logical(out$truefalse)
+
+  checkbox_col_types <- map(out %>% select(contains("checkbox")), class)
```

Comment:
Ugghhhh... can you make this less complicated? I really don't know what's going on here. There should really be no `for` loops or `map` functions in tests.

Reply:
Haha I knew you'd call this out. I was trying to make an elegant way of capturing all `checkbox_*` fields to test, but if you prefer we can either test one field or do many single tests. I think in the `clean_*` tests we just tested a single `checkbox_*` field.